common4j/src/main/com/microsoft/identity/common/java/logging/Logger.java

```diff
@@ -417,6 +487,51 @@ public void run() {
         });
     }
 
+    /**
+     * Temporary method to allow the project to compile, without having to make changes everywhere
+     */
```

Comment:
Is this comment / method still relevant with the complete changes?

Reply:
Good catch. Updated.